includes/connection/class-product-attributes.php

```diff
@@ -59,13 +65,20 @@ public static function register_connections() {
 	public static function get_connection_config( $args = [] ): array {
 		return array_merge(
 			[
-				'fromType'       => 'Product',
-				'toType'         => 'ProductAttribute',
-				'fromFieldName'  => 'attributes',
-				'connectionArgs' => self::get_connection_args(),
-				'resolve'        => function ( $source, array $args, AppContext $context, ResolveInfo $info ) {
+				'fromType'      => 'Product',
+				'toType'        => 'ProductAttribute',
+				'fromFieldName' => 'attributes',
+				'resolve'       => function ( $source, array $args, AppContext $context, ResolveInfo $info ) {
 					$resolver = new Product_Attribute_Connection_Resolver();
-
+					// phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					switch ( $info->fieldName ) {
+						case 'globalAttributes':
+							$args['where']['type'] = 'global';
+							break;
+						case 'localAttributes':
+							$args['where']['type'] = 'local';
+							break;
+					}
```

Comment:
Why not put this in the connection resolver's `get_query_args()` function?

Reply:
Yes. I will create a connection resolver for this on a different task, but for now, I think it's important to merge this PR as it fixes a critical error.